frappe/public/js/frappe/list/base_list.js

```diff
@@ -380,9 +378,11 @@ frappe.views.BaseList = class BaseList {
 			// render
 			this.prepare_data(r);
 			this.toggle_result_area();
+			this.toggle_paging && this.$paging_area.toggle(false);
```

Comment:
If this is supposed to be run only for list_view, then we should move this code to list_view.js. We have `before_render` and `after_render`